test/tests/userobjects/element_subdomain_modifier/block_restricted.i

```diff
@@ -55,7 +55,7 @@
     type = FunctionAux
     variable = phi
     function = moving_circle
-    execute_on = 'INITIAL TIMESTEP_BEGIN TIMESTEP_END'
+    execute_on = 'INITIAL TIMESTEP_BEGIN'
```

Comment:
So if I understand this correctly, we had to run these all on timestep_end because whatever they did at timestep_begin got overwritten, but you don't need to do that anymore. Is that right?

You do still have to re-gold a bunch of files, though, so something else must have changed as well.

Reply:
I believe the other regolds are for `INITIAL`, in a good way.

Reply:
Before it was ran with TIMESTEP_END, this hid the error of the overwriting except for INITIAL. Since INITIAL was messed up, I had to regold. I dropped the TIMESTEP_END so that the test wouldn't overwrite the variable after the UserObjects is performed so that later if the code has another bug the output will show it.